### PR TITLE
Move company credit page and store credit page to the customer block

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request moves the company credit page and the stores credit page to the customer block. The company credit page and the store credit page are related to the customer block, not sales.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/sales/store-credit.html
https://docs.magento.com/user-guide/sales/credit-configure.html
https://docs.magento.com/user-guide/sales/store-credit-using.html
https://docs.magento.com/user-guide/sales/refunds-customer-account.html
https://docs.magento.com/user-guide/sales/credit-company.html
https://docs.magento.com/user-guide/sales/credit-company-activity.html
https://docs.magento.com/user-guide/sales/credit-company-reimburse.html

### CODE REVIEW

**Comment by leslietilling** [comment]
All renamed files need redirects for the URLs that are currently published. I can do this if needed.

**Comment by dobooth** [comment]
@ZhovnirMaria looks like some merge conflicts here...

**Comment by ZhovnirMaria** [comment]
@dobooth Done, please check if I missed something😊

**Comment by leslietilling** [comment]
@ZhovnirMaria  We still need to address the redirects for renamed files. If you are unclear on how that should work I can do the commit.

**Comment by leslietilling** [comment]
@ZhovnirMaria  We still need to address the redirects for renamed files. If you are unclear on how that should work I can do the commit.

**Comment by ZhovnirMaria** [comment]
@leslietilling, hi)
if you want, you can do it by yourself or please describe how I can do it😊
Thanks!

**Comment by leslietilling** [comment]
> @leslietilling, hi)
> if you want, you can do it by yourself or please describe how I can do it😊
> Thanks!

I added a new page to our contributor guide: https://github.com/magento/merchdocs/wiki/Topic-Redirects

Let me know if you have any questions.

**Comment by ZhovnirMaria** [comment]
@leslietilling hi, I added a few changes, please check🙏

File: src/customers/credit-company.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ---
 b2b_only: true
 title: Company Credit
-redirect_from:
```
Comment (leslietilling) [inline src/customers/credit-company.md:4]:
```suggestion
redirect_from:
  - /sales/credit-company.html
```

File: src/customers/store-credit.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 ee_only: true
 title: Store Credit
 redirect_from:
-  - /sales/store-credit-workflow.html
+  - /sales/store-credit.html
```
Comment (leslietilling) [inline src/customers/store-credit.md:5]: Let's leave the existing redirect along with the new one. I will update the contributor guide wiki page to add this use case. The system does support more than one redirect.

```suggestion
  - /sales/store-credit-workflow.html
  - /sales/store-credit.html
```

File: src/customers/credit-company-activity.md [hunk 3]
```diff
@@ -1,8 +1,6 @@
 ---
 b2b_only: true
 title: Company Credit Activity
-redirect_from:
```
Comment (leslietilling) [inline src/customers/credit-company-activity.md:4]: Let's keep the existing redirects, which are a result of some previous movement.

```suggestion
redirect_from:
  - /sales/credit-company-activity.html
 ```

File: src/customers/credit-company-reimburse.md [hunk 4]
```diff
@@ -1,8 +1,6 @@
 ---
 b2b_only: true
 title: Receiving Payments
-redirect_from:
```
Comment (leslietilling) [inline src/customers/credit-company-reimburse.md:4]:
```suggestion
redirect_from:
  - /sales/credit-company-reimburse.html
 ```

**Review by leslietilling — APPROVED** [review]
I'll commit my suggestions for leaving the existing redirects. Thanks for improving the org here :)

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**